### PR TITLE
Add osu! South East Asia Tournament 3

### PR DESCRIPTION
**Pull request by fajar13k**
- Add oSEAT3 Article
- Some groups might be messed, some participants FF unexpectedly

---

### CODE REVIEW

**Comment by cl8n** [comment]
is it possible to get a 4man bracket image without the mouse pointer

**Comment by fajar13k** [comment]
Replaced

File: wiki/Tournaments/oSEAT/3/en.md [hunk 1]
```diff
@@ -0,0 +1,616 @@
...
+| Streamer | ![][flag_ID] [Fre](https://osu.ppy.sh/users/3484548), ![][flag_ID] [Skydiver](https://osu.ppy.sh/users/4750008) |
+| Commentator | ![][flag_ID] [Fre](https://osu.ppy.sh/users/3484548), ![][flag_VN] [Hoaq](https://osu.ppy.sh/users/7696512), ![][flag_ID] [Skydiver](https://osu.ppy.sh/users/4750008) |
+| Designer | ![][flag_SG] [youi](https://osu.ppy.sh/users/7537133) |
+| Statistician | ![][flag_VN] [steve\_04\_](https://osu.ppy.sh/users/10852911) |
+| Wiki editor | ![][flag_ID] [fajar13k](https://osu.ppy.sh/users/7100002) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/978343)
+- [BTT Discord server](https://discord.gg/sVHndPU)
```
Comment (cl8n) [inline wiki/Tournaments/oSEAT/3/en.md]: what does btt stand for / is this the right link

File: wiki/Tournaments/oSEAT/3/en.md [hunk 2]
```diff
@@ -0,0 +1,616 @@
...
+#### Grand Finals
+
+1. Matches will be best of 13 (first to 7 points)
+2. Mappool for respective rounds will contain 5 NM, 3 HD, 3 HR, 4 DT, and 4 FM maps.
+3. There will be a bracket reset match, if necessary.
+
+### Late penalties
+
+1. **Less than 5 minutes**: no penalty.
+2. **5-10 minutes**: No warmup for the respective player, opposing player chooses whether to pick first or ban first. *Note: no more substitution rule for groups. Players will only be subbed out of groups if they request to drop out before their first match*
```
Comment (cl8n) [inline wiki/Tournaments/oSEAT/3/en.md]: this note seems completely unrelated to the rest of the line
